Add Sitecore

MartinMiles opened the pull request:
**https://github.com/MartinMiles/Awesome-Sitecore**

**A missing list for such a top-rated CMS  as Sitecore**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

cdosoftei commented:
Thanks for the submission!

There are a number of issues with this submission, here are the top ones:
* The repository hasn't been around for too long (it has to be at least 30 days old);
* ToC's naming is not in conformance (should be named Contents, not Table of Contents)
www;
* Missing Sitecore logo;
* Some link descriptions are not showing consistent formatting and proper spelling/grammar.

MartinMiles commented:
Added logo, made all the links aligned with the requirements (dots, cases, spaces, more), got rid of duplicated and spell checks, added license and contribution, few more minors

MartinMiles commented:
unicorn

msmolyakov commented:
Please move your entry to the bottom of the category.

Requirement:
> Your entry should be added at the bottom of the appropriate category.

msmolyakov commented on readme.md:
```diff
@@ -502,6 +502,7 @@
 
 ## Content Management Systems
 
+- [Sitecore](https://github.com/MartinMiles/Awesome-Sitecore)
```
Please move your entry to the bottom of the category.

Requirement:
> Your entry should be added at the bottom of the appropriate category.

MartinMiles replied:
Done

MartinMiles commented:
> * ToC's naming is not in conformance (should be named Contents, not Table of Contents)
> * Missing Sitecore logo;
> * Some link descriptions are not showing consistent formatting and proper spelling/grammar.

The above is done.

mxschmitt commented:
The submission looks fine for me.

MartinMiles commented:
So, 30 days have passed. The list complies with the guidance. Linter also passes well
Please feel free to merge into main repository

sindresorhus commented on readme.md:
```diff
@@ -508,6 +508,7 @@
 - [Textpattern](https://github.com/drmonkeyninja/awesome-textpattern#readme) - Lightweight PHP-based CMS.
 - [Drupal](https://github.com/nirgn975/awesome-drupal#readme) - Extensible PHP-based CMS.
 - [Craft CMS](https://github.com/craftcms/awesome#readme) - Content-first CMS.
+- [Sitecore](https://github.com/MartinMiles/Awesome-Sitecore) - Sitecore Experience Platform.
```
> Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
✅ - [iOS](…) - Mobile operating system for Apple phones and tablets.
✅ - [Framer](…) - Prototyping interactive UI designs.
❌ - [iOS](…) - Resources and tools for iOS development.
❌ - [Framer](…)
❌ - [Framer](…) - prototyping interactive UI designs

You also need to add `#readme`

sindresorhus commented:
> Includes a succinct description of the project/theme at the top of the readme. (Example)
✅ Mobile operating system for Apple phones and tablets.
✅ Prototyping interactive UI designs.
❌ Resources and tools for iOS development.
❌ Awesome Framer packages and tools.

MartinMiles commented:
@sindresorhus renamed the entry as per your request. Now looking explanatory.

MartinMiles commented:
Looking forward for the merge, @sindresorhus please let me know if there's anything else left to get  changed

sindresorhus commented on readme.md:
```diff
@@ -508,6 +508,7 @@
 - [Textpattern](https://github.com/drmonkeyninja/awesome-textpattern#readme) - Lightweight PHP-based CMS.
 - [Drupal](https://github.com/nirgn975/awesome-drupal#readme) - Extensible PHP-based CMS.
 - [Craft CMS](https://github.com/craftcms/awesome#readme) - Content-first CMS.
+- [Sitecore](https://github.com/MartinMiles/Awesome-Sitecore) - A robust digital marketing system that combines CMS with contextual intelligence and omnichannel automation technologies. It also offers a rich variety of tools out of the box for emailing, analytics, campaigns and marketing automation tools for managing website(s).
```
This is too verbose. See the other descriptions for inspiration.

MartinMiles replied:
> This is too verbose. See the other descriptions for inspiration.

Provided less verbose description.

sindresorhus commented:
You did not fully read my comment in https://github.com/sindresorhus/awesome/pull/1747#discussion_r415864790

sindresorhus commented:
https://github.com/sindresorhus/awesome/pull/1747#issuecomment-620025865 is not resolved.

sindresorhus commented:
You cannot just copy-paste the contributing guidelines for Awesome: https://github.com/MartinMiles/awesome-sitecore/blob/master/contributing.md

sindresorhus commented:
Run https://github.com/sindresorhus/awesome-lint on your list and ensure it passes.

MartinMiles commented:
Linter passes well and returns no errors other than "remark-lint:no-dead-urls" which gives a false negative for the majority of links  from the list, claiming these links are dead. In fact, they are **not** dead and that could be easily prove by opening any of them. Something might be wrong with awesome-lint tool

MartinMiles commented:
> You cannot just copy-paste the contributing guidelines for Awesome: https://github.com/MartinMiles/awesome-sitecore/blob/master/contributing.md

Of course (silly me!). That has now been fixed by providing the relevant guidance for awesome-sitecore list

MartinMiles commented:
@sindresorhus , all 3 issues have been fixed:

1. Description of the list entry changed, for the relevancy.
2. Contributing.md provided as it should be.
3. Linter passes well (apart from falsely claimed dead URLs)

MartinMiles commented:
@sindresorhus do you mind merging my list, unless there is something else left for me to do?

sindresorhus commented:
> Includes a succinct description of the project/theme at the top of the readme.

MartinMiles commented:
> > Includes a succinct description of the project/theme at the top of the readme.

Done

sindresorhus commented:
In `Sitecore is a digital platform that helps managing multiple websites from one place.`, linkify `Sitecore` to its website.

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1267520948942417920
